bootstrap-hover-dropdown.js

```diff
@@ -8,7 +8,11 @@
  * Description: A simple plugin to enable Bootstrap dropdowns to active on hover and provide a nice user experience.
  * License: MIT
  * Homepage: http://cameronspear.com/blog/bootstrap-dropdown-on-hover-plugin/
+
+
  */
+
+
```

Comment:
Looks like some blank lines snuck in!
```suggestion
 */
```